apps/laboratory/tests/shared/utils/email.ts

```diff
@@ -78,8 +77,6 @@ export class Email {
   }
 
   getEmailAddressToUse(index: number, domain = EMAIL_DOMAIN): string {
-    const prefix = randomBytes(12).toString('hex')
-
-    return `${prefix}-w${index}@${domain}`
+    return `w3m-w${index}@${domain}`
```

Comment:
Wouldn't this result in same runner executing in parallel in different contexts? 
eg. multiple prs here or on secure site, or local dev running tests

unless it's an accepted "risk" due to cost?

Reply:
@tomiir It would. Previous change was implemented for that exact reason. And you're right, due to cost its reverted.